test/extended/testdata/templates/templateservicebroker_bind.yaml

```diff
@@ -59,6 +61,31 @@ objects:
           to:
             kind: Service
             name: service
+      - apiVersion: apiextensions.k8s.io/v1beta1
+        kind: CustomResourceDefinition
```

Comment:
until the cluster-scoped->namespaced and cross-namespace ownerRef issues are settled, the TSB shouldn't set up objects that result in those invalid relationships

Reply:
I actually wanted to put code in place to prevent those relationships (the TSB will happily set them up now, and they don't work, and I don't anticipate ownerRef API changes to support them in 3.10 timeframe)